### PR TITLE
Make sure to not render overlay when not open.

### PR DESCRIPTION
**Pull request by psachs21**

### CODE REVIEW

File: src/popover/index.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ class Popover extends Component {
   }
 
   _getBackDropElement = () => {
-    if (!this.props.onClickOut) return null;
+    if (!this.props.onClickOut || !this.props.isOpen) return null;
```
Comment (nickfaulkner) [inline src/popover/index.js]: won't this add it regardless if there is a click out function passed or not?

shouldn't it be something like this:

!this.props.onClickOut || this.props.onClickOut && !this.props.isOpen

Comment (psachs21) [inline src/popover/index.js]: How? If no onClickOut is provided, the condition passes (!this.props.onClickOut) and it returns null. The only time this condition will pass to render the overlay is when there is a click handler specified and it's open. Arguably i could reorient the question but the logic is sound.

Comment (nickfaulkner) [inline src/popover/index.js]: my bad... one of those days.

Comment (SeeThruHead) [inline src/popover/index.js]: yeah negative or guards can be confusing

maybe just

if (clickout && isOpen) return (
  stuff
);

??

**Pull request merged**